src/test/java/com/google/api/codegen/testdata/nodejs_library.baseline

```diff
@@ -710,6 +726,7 @@ LibraryServiceApi.prototype.moveBook = function moveBook() {
  *
  * @param {?Object} otherArgs
  * @param {String} otherArgs.name
+ * @param {number} otherArgs.pageSize
```

Comment:
In the GAPIC config, this field is deliberately not configured to be part of page streaming. This is supposed to model the fact that the config distinguishes the semantics of the `page_size` field from the field name itself.